src/instrumentation/helpers/trace-serializer.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2024 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const DEFAULT_IGNORE_KEYS = ["emitter", "logger", "tokens", "createdBy", "client"];
```

Comment:
I would bind this property to the function directly.

`traceSerializer.ignoredKeys` so user can easily modify it.

Reply:
I prefer to have 1 way, how to configure the instrumentation. You can modify the ignored keys via the env variable `BEE_FRAMEWORK_INSTRUMENTATION_IGNORED_KEYS` right now.  The instrumentation enabling/disabling is via the env variable as well. 

```
BEE_FRAMEWORK_INSTRUMENTATION_ENABLED=true
BEE_FRAMEWORK_INSTRUMENTATION_IGNORED_KEYS=apiKey,token
```